staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/apiserver.go

```diff
@@ -172,13 +172,17 @@ func (c completedConfig) New(delegationTarget genericapiserver.DelegationTarget)
 	}
 
 	versionDiscoveryHandler := &versionDiscoveryHandler{
-		discovery: map[schema.GroupVersion]*discovery.APIVersionHandler{},
-		delegate:  delegateHandler,
+		discoveryStorage: atomic.Value{},
```

Comment:
isn't `atomic.Value{}` the default null value? Then we don't need this initializer.

Reply:
Yes, removed.